src/main/webapp/WEB-INF/tags/head.tag

```diff
@@ -1,2 +1,3 @@
 <%@tag description="Common head content" pageEncoding="UTF-8"%>
-<!-- This is where common <link> and <script> tags would go. -->
+<link href="../../style.css" rel="stylesheet" type="text/css">
+<script async src="https://cse.google.com/cse.js?cx=017022763452466193978:ejddswiptcs"></script>
```

Comment:
Is it safe for this script to be async? I assume it's looking for the gcse-searchbox-only class and so forth, which if the script is fetched asynchronously, you could run into a race condition where the rest of the doc isn't parsed yet.